### PR TITLE
New package: upwork-5.1.0.562

### PR DESCRIPTION
**Pull request by ap4y**
Desktop client for [Upwork](https://www.upwork.com).

### CODE REVIEW

File: srcpkgs/upwork/files/LICENSE [hunk 1]
```diff
@@ -0,0 +1,30 @@
+End-User License Agreement
```
[the remark this answers is not in the document]
Comment (ap4y) [reply srcpkgs/upwork/files/LICENSE]: Afaik upwork shows license on startup but it's not bundled into deb package. I couldn't re-trigger license UI from within the app so  I copied license from the [AUR package](https://aur.archlinux.org/cgit/aur.git/tree/LICENSE?h=upwork). I found it on the [upwork.com](https://www.upwork.com/legal#softwarelicense) a bit later, Team App is a previous name of the Upwork Desktop.

[the remark this answers is not in the document]
Comment (cr6git) [reply srcpkgs/upwork/files/LICENSE]: Then you could download the pdf license from [upwork.com](https://www.upwork.com/legal#softwarelicense)
````
distfiles="https://updates-desktopapp.upwork.com/binaries/v${_rawver}_${_hashver}/upwork_${version}_amd64.deb
 https://upwork.pactsafe.io/versions/5ab3efef63d65b6a18aab106.pdf>LICENSE.pdf"
checksum="af70dae3d1cd5910d3ab591201caf75b13758d12e487b44cd32452c305df68c0
 0949da5999c3b948bcc4165e6c55522915d9bab1790d27120eda0915aae143bb"

do_install() {
	vlicense ${XBPS_SRCDISTDIR}/${pkgname}-${version}/LICENSE.pdf
	...
````

[the remark this answers is not in the document]
Comment (ap4y) [reply srcpkgs/upwork/files/LICENSE]: Thanks! Should be fixed now.

**Comment by ap4y** [comment]
Test errors are not related to the package.

**Comment by cr6git** [comment]
I slightly reworked your template.

**Comment by ap4y** [comment]
Thanks!

**Pull request closed without merging**